src/app/main.js

```diff
@@ -5,81 +5,37 @@ define([
   'dojo/dom-style',
   'dojo/dom-attr',
 
-  'esri/layers/FeatureLayer',
-  'esri/InfoTemplate',
   'esri/graphic',
-  'esri/dijit/Geocoder',
-  'esri/dijit/LocateButton',
-  'esri/dijit/Legend',
 
   'bootstrap-map-js/js/bootstrapmap',
```

Comment:
You should be able to remove this from both main.js and dashboard.html now that it's in map.js, right? If so, please do.